plugins/talk-plugin-auth/client/login/containers/SignIn.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect, withSignIn } from 'plugin-api/beta/client/hocs';
+import { compose } from 'recompose';
+import SignIn from '../components/SignIn';
+import { bindActionCreators } from 'redux';
+import * as views from '../enums/views';
+import { setView, setEmail, setPassword } from '../actions';
+
+class SignInContainer extends Component {
+  state = {
+    recaptchaResponse: '',
```

Comment:
Shouldn't the empty response be `null`?

Reply:
Question back, why not ''?

Reply:
Based on your previous recommendations, the "empty" state of a string is `null`, whereas `''` is a valid value, while falsy (I think?) isn't that clear.

Reply:
I guess you are right